Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -736,7 +728,7 @@ class BrowserViewController: UIViewController {
         }
         
         // The user either skipped or didn't complete onboarding.
-        let isRewardsEnabled = rewards?.ledger.isEnabled == true
+        let isRewardsEnabled = rewards.ledger.isEnabled == true
```

Comment:
small nit: the == true is no longer needed since we aren't comparing optionals, can just be `rewards.ledger.isEnabled`, or even remove line and replace usages with `rewards.ledger.isEnabled` like I see already done below